decide/test_rust.py

```diff
@@ -0,0 +1,7 @@
+import rust
+
+decider = rust.init("decision_maker_name", "cfg.json")
```

Comment:
we might want to make a minimal cfg.json rather than the one you included.

Reply:
will do!